iceoryx_posh/include/iceoryx_posh/internal/popo/building_blocks/chunk_receiver.hpp

```diff
@@ -36,10 +36,11 @@ enum class ChunkReceiveError
 /// The
 /// ChunkRceiver holds the ownership of the SharedChunks and does a bookkeeping which chunks are currently passed to the
 /// user side.
-class ChunkReceiver : public ChunkQueuePopper
+template <typename ChunkQueueDataType>
```

Comment:
shouldn't this be ChunkQueueType here to be symmetric?

ChunkReceiverData has ChunkQueueDataType as template and then ChunkReceiver has the ChunkQueueType

OK, then you would inherit from something like ChunkQueuePopper<ChunkQueueType::ChunkQueueData_t>

But I feel this would be more one layer uses the one below

Reply:
There's no `ChunkQueue` right now. There's only the queue popper. Should we rework the structure in EA?

Reply:
Right, there is no spoon. It must be ChunkQueuePopper

Reply:
Done